file/main.fmf

```diff
@@ -2,4 +2,7 @@ summary: Example of a beakerlib library
 contact: Petr Šplíchal <psplicha@redhat.com>
 test: ./test.sh
 framework: beakerlib
-require: coreutils
+require:
+  - coreutils
+  - pattern: random_file.sh
```

Comment:
Just to clarify: In the spec we have:

> When referencing local files or directories use `type: file` and define list of paths relative to the fmf root directory.

So we don't expect `/scripts/random_file.sh` here but we plan to support matching the filename anywhere in the path, right?

Reply:
Well it's a regex so anything goes, full relative path is not necessary. Maybe the description is a bit off.

Reply:
Ok, just wanted to clarify the intention.